src/components/ImageSlider/ImageSlider.js

```diff
@@ -0,0 +1,110 @@
+import PropTypes from 'prop-types';
+import React, { useState } from 'react';
+import { css } from '@emotion/react';
+import { Button, Icon } from '@newrelic/gatsby-theme-newrelic';
+
+const ImageSlider = ({ images, height }) => {
+  const [selectedImageIndex, setSelectedImageIndex] = useState(0);
+
+  const handleClickNext = () => {
+    const nextImageIndex = selectedImageIndex + 1;
+    setSelectedImageIndex(nextImageIndex % images.length);
+  };
+
+  const handleClickPrev = () => {
+    const prevImageIndex = selectedImageIndex - 1;
+    if (prevImageIndex < 0) {
+      setSelectedImageIndex(images.length - 1);
+    } else {
+      setSelectedImageIndex(prevImageIndex % images.length);
+    }
+  };
+
+  return (
+    <div
+      css={css`
+        position: relative;
+        margin-bottom: 2rem;
+      `}
+    >
+      {images.length > 1 && (
+        <>
+          <Button
+            onClick={handleClickPrev}
+            variant={Button.VARIANT.PLAIN}
+            css={css`
+              position: absolute;
+              top: 38%;
+              left: 0;
+              background: none;
+              color: var(--color-white);
+              border: none;
+              cursor: pointer;
```

Comment:
Totally optional, but you could probably cut down on a lot of the duplicate code by adding the `button` styles to the parent `div`. I think the only style that's unique to these buttons is the `left` / `right`.

Reply:
that parent `<div>` is for the entire slider container, so just added another `div` for these arrows and made them flex instead